lib/src/formatters/float_formatter.dart

```diff
@@ -93,7 +93,7 @@ class FloatFormatter extends Formatter {
   }
 
   @override
-  String asString() {
+  String? asString() {
```

Comment:
This change isn't needed. The `_output` var has a direct null check before it's returned, so the line can be changed to ` return _output!;`